cloud-aws-native/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNativeResourceConnector.java

```diff
@@ -96,6 +96,11 @@ public String getDBStackTemplate(DatabaseStack databaseStack) throws TemplatingN
         throw new TemplatingNotSupportedException();
     }
 
+    @Override
+    public void updateDiskVolumes(AuthenticatedContext authenticatedContext, List<String> volumeIds, String diskType, int size) throws Exception {
+        throw new UnsupportedOperationException("Method not implemented.");
```

Comment:
do you have a CB ticket for this? or we won't support native?

Reply:
i have implemented this. thanks for pointing it out